app/assets/javascripts/tests/_mobile_sessions.test.js

```diff
@@ -295,6 +295,20 @@ test('hasSelectedSessions with selected session returns true', t => {
   t.end();
 });
 
+test('drawSessionsInLocation calls drawSession.drawMobileSessionStartPoint', t => {
+  const session = "session"
+  const sessions = { session1: session }
+  const sessionsUtils = { get: () => (sessions) };
```

Comment:
You can drop the `( )` in `get: () => (sessions)`. You need them only when it's `get: () => ({sessions})`